pepys_import/resolvers/command_line_resolver.py

```diff
@@ -673,14 +444,16 @@ def add_to_platforms(
         :param privacy: Name of :class:`Privacy`
         :type privacy: Privacy
         :param change_id: ID of the :class:`Change` object
-        :type change_id: Integer or UUID
+        :type change_id: UUID
         :return:
         """
         print("Ok, adding new platform.")
         if platform_name:
             platform_name = prompt("Please enter a name: ", default=platform_name)
+            trigraph = None
             if len(platform_name) >= 3:
                 trigraph = prompt("Please enter trigraph (optional): ", default=platform_name[:3])
+            quadgraph = None
```

Comment:
Do we really want to make it so we don't ask them for a trigraph/quadgraph if the platform name is shorter than 3/4. Can trigraphs/quadgraphs exist in these circumstances @IanMayo?

Reply:
Thanks @robintw  - good question.

Yes, they can be shorter than 4 chars: https://en.wikipedia.org/wiki/ICGV_%C3%9E%C3%B3r

(I've just learned something new about uniqueness - I'll add it to that issue)

Reply:
Solved. Trigraph/quadgraph are asked even though platform name is less than 3/4 characters:
https://github.com/debrief/pepys-import/pull/408/commits/e2c48428d09b4765560facd81a944f4df2620c79